test/extensions/filters/http/wasm/config_test.cc

```diff
@@ -211,6 +211,7 @@ TEST_P(WasmFilterConfigTest, YamlLoadFromRemoteWASM) {
   cb(filter_callback);
 }
 
+#if 0
 TEST_P(WasmFilterConfigTest, YamlLoadFromRemoteWASMFailOnUncachedThenSucceed) {
```

Comment:
Why is this test disabled?

Reply:
The tests are mocks,  so they are heavily dependent on the details of the implementation.  They also didn't actually provide any confidence since the tests passed and the implementation failed on integration because, well mocks are not real tests IMO.  The integration test gives more confidence than these tests.   That said I am updating them in envoy-wasm and will backport them here when I get them done.

Reply:
The tests are working.  Reactivated them.